meta/sai_rpc_frontend.cpp

```diff
@@ -929,6 +929,16 @@ int start_p4_sai_thrift_rpc_server(char *port) {
   return status;
 }
 
+/**
+ * @brief Start Thrift RPC server Wrapper
+ */
+int start_sai_thrift_rpc_server(int port)
+{
+    static char port_str[10];
+    snprintf(port_str, sizeof(port_str), "%d", port);
```

Comment:
just wondering why actually port is passed via string ? it make no sense, it should be uint16_t